datareservoirio/client.py

```diff
@@ -255,8 +261,11 @@ def get(
             start time (inclusive) of the series given as anything
             pandas.to_datetime is able to parse.
         end : optional
-            stop time (inclusive) of the series given as anything
+            stop time of the series given as anything
             pandas.to_datetime is able to parse.
+        end_inclusive : optional
+            If True (default), the stop time is inclusive
+            If False, the stop time is exclusive
```

Comment:
We decided to not give the user an option. Make the client always non-inclusive. I have updated the story title in Jira.